ee/vellum_ee/workflows/display/workflows/base_workflow_display.py

```diff
@@ -844,10 +844,57 @@ def serialize_module(
             client=client,
             dry_run=dry_run,
         )
+
+        additional_files = BaseWorkflowDisplay._gather_additional_module_files(module)
+
+        exec_config = workflow_display.serialize()
```

Comment:
Update the `serialize` method to define a private attribute called `_serialized_files`. For now, make it a constant array with the following:
```
[
"nodes/*",
"workflow.py",
"state.py",
"inputs.py",
"display/*",
"__init__.py"
]
```